src/task/CamelKTaskDefinition.ts

```diff
@@ -58,7 +60,7 @@ export class CamelKTaskProvider implements vscode.TaskProvider {
 		this.tasks = [];
 		let taskDefinition = {
 			"type": CamelKTaskProvider.START_CAMELK_TYPE,
-			"label": "Start in dev mode Camel K integration opened in active editor",
+			"label": TASK_NAME_DEFAULT_DEPLOY_FROM_ACTIVE_EDITOR,
```

Comment:
The name of the constant does not really fit what it does when reading the label of that constant. Maybe we can improve that a bit?